sound/soc/sof/intel/hda-loader.c

```diff
@@ -414,12 +413,12 @@ static int cl_dsp_init_skl(struct snd_sof_dev *sdev)
 				HDA_DSP_ADSPIC_IPC, HDA_DSP_ADSPIC_IPC);
 
 	/* enable IPC DONE interrupt */
-	snd_sof_dsp_update_bits(sdev, HDA_DSP_BAR, chip->ipc_ctl,
+	snd_sof_dsp_update_bits(sdev, HDA_DSP_BAR, HDA_DSP_REG_HIPCCTL,
 				HDA_DSP_REG_HIPCCTL_DONE,
 				HDA_DSP_REG_HIPCCTL_DONE);
 
 	/* enable IPC BUSY interrupt */
-	snd_sof_dsp_update_bits(sdev, HDA_DSP_BAR, chip->ipc_ctl,
+	snd_sof_dsp_update_bits(sdev, HDA_DSP_BAR, HDA_DSP_REG_HIPCCTL,
```

Comment:
I think we can still use chip->ipc_ctl here to make the code more comment with cl_dsp_init()?